### ISSUE TITLE
I was asked to report why I prefered Oruxmaps to rmaps simply for display of O.S. maps

### ISSUE THREAD

**Issue by GoogleCodeExporter** [RobertDeveloper/RMaps.0.9.x#132]
```
I am using the latest Android apps on a Samsung Galaxy 10.1

This is the highest magnification of rmaps that still shows footpaths. 

http://i1283.photobucket.com/albums/a544/satipera/Screenshot_2013-08-30-10-55-41
_zpsf3a96708.png

If the magnification is increased again then this is the image

http://i1283.photobucket.com/albums/a544/satipera/Screenshot_2013-08-30-10-55-15
_zps21479c78.png

It is clear to see that footpaths are no longer shown and so the first image is 
as good as it gets for rmaps using Ordnance Survey.

This is the first level of magnification on Oruxmaps that shows field outlines 
which rmaps does not do. When you are in a field in the middle of nowhere with 
no style where it should be or apparent footpath the field outline on the map 
is very useful.

http://i1283.photobucket.com/albums/a544/satipera/Screenshot_2013-08-30-10-59-59
_zps5e5beab1.png

In addition the magnification can be increased even further than this

http://i1283.photobucket.com/albums/a544/satipera/Screenshot_2013-08-30-11-02-56
_zps37186acd.png

So to sum up. For the additional mapping information and further magnification 
that still shows footpaths, bridleways etc I prefer Oruxmaps. If I could do 
this on rmaps I would use it as I would prefer to use FLOSS.
```

Original issue reported on code.google.com by `satipera@googlemail.com` on 30 Aug 2013 at 1:50
